kemdk/src/main/java/dk/gls/kemdk/KotlinEMDK.kt

```diff
@@ -0,0 +1,44 @@
+package dk.gls.kemdk
+
+import android.content.Context
+import dk.gls.kemdk.deviceInformation.DeviceInformation
+import dk.gls.kemdk.deviceInformation.IDeviceInformation
+import dk.gls.kemdk.emdk.EMDKIntegration
+import dk.gls.kemdk.emdk.EMDKThrowable
+import dk.gls.kemdk.model.OEMInfo
+import dk.gls.kemdk.model.RetryConfiguration
+import dk.gls.kemdk.oemInfo.OEMInfoRequester
+import dk.gls.kemdk.oemInfo.OEMInfoThrowable
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+
+open class KotlinEMDK : IKotlinEMDK {
```

Comment:
Not that we should change it, but I think it is a bit weird that class names include the language they are written in

Reply:
But it is the library name? 😅